src/main/scala/no/ndla/learningpathapi/controller/LearningpathControllerV2.scala

```diff
@@ -161,7 +161,7 @@ trait LearningpathControllerV2 {
       val language = paramOrDefault(this.language.paramName, Language.AllLanguages)
       val id = long(this.learningpathId.paramName)
 
-      readService.withIdV2(id, language, userOrClientId) match {
+      readService.withIdV2(id, language, AuthUser.get) match {
```

Comment:
Bruke `requireUserId` for å få med feilhåndteringen?

Reply:
Det drar egentlig med seg en større refaktoreringsjobb, siden `readService.withIdV2` forventer en `Option[String]` og ikke en `String`. Access rettigheter er håndtert lengre "inn" i koden slik det er nå (man får fremdeles en "access denied" feilmelding).

Kan selvfølgelig gjøre det, men vet ikke hvor viktig det er

Reply:
Åh. Jeg trodde det var slik før du slettet, for så ikke noen direkte AuthUser.get der?

Reply:
Neida, var `AuthUser.get` før jeg endret også: https://github.com/NDLANO/learningpath-api/commit/d6d8c61afb2d082436c80cd95735dcf42f880371 :)

Reply:
Ah. Må ha sett på feil commit jeg da. :$ Da er jeg enig...